style_guides/js_style_guide.md

```diff
@@ -83,19 +25,6 @@ const foo = `Hello, ${name}`;
 const foo = 'Hello, ' + name;
 ```
 
-## Use template strings to avoid escaping single quotes
```

Comment:
:information_source: Actually in conflict with our current Prettier rules, so I plainly removed this, since I think prettier overwrites styleguide right now.